encrypted-config-value-module/src/main/java/com/palantir/config/crypto/jackson/EncryptedConfigMapperUtils.java

```diff
@@ -37,4 +37,12 @@ private EncryptedConfigMapperUtils() {}
         return mapper.treeToValue(substitutedNode, clazz);
     }
 
+    public static <T> T getConfig(String configFileContent, Class<T> clazz, ObjectMapper mapper)
+            throws JsonParseException, JsonMappingException, IOException {
+        JsonNode configNode = mapper.readValue(configFileContent, JsonNode.class);
+        JsonNode substitutedNode = JsonNodeVisitors.dispatch(
+                mapper.valueToTree(configNode),
```

Comment:
valueToTree takes ? -> JsonNode... why do we call it when the input is already a JsonNode?

Reply:
That does sound unnecessary